tchannel/thrift/client.py

```diff
@@ -75,15 +78,21 @@ def post_comment(articleId, msg, hostport=None):
 
     method_names = get_service_methods(service_module.Iface)
 
-    def new(cls, tchannel, hostport=None, trace=False):
+    def new(cls, tchannel, hostport=None, trace=False, protocol_headers=None):
         """
         :param tchannel:
             TChannel through which the requests will be sent.
         :param hostport:
             Address of the machine to which the requests will be sent, or None
             if the TChannel will do peer selection on a per-request basis.
+        :param trace:
+            Flag to turn on/off trace in tchannel.
+        :param protocol_headers:
+            Protocol headers of tchannel.
```

Comment:
For `trace`,

    Whether Zipkin tracing is enabled.

For `protocol_headers`

    Protocol-level headers to send with the request.